### DISCUSSION TITLE
Fe with Quantum espresso + wannier90

### DISCUSSION THREAD

**Discussion by chelman1** [mailhexu/TB2J#14] [General]
Hi, I just calculate the Fe Bcc case with wannier90. Bands and spread seems to be fine.
However, the interaction are very different for first neighbors. I copy the table.
Any idea? thanks!
Exchange:
    i      j          R        J_iso(meV)          vector          distance(A)
----------------------------------------------------------------------------------------
   Fe1   Fe1   ( -1,   0,   0) -9.9017   (-1.435, -1.435, -1.435)  2.485
J_iso: -9.9017
----------------------------------------------------------------------------------------
   Fe1   Fe1   ( -1,   1,  -1) -12.5034   (-1.435,  1.435, -1.435)  2.485
J_iso: -12.5034
----------------------------------------------------------------------------------------
   Fe1   Fe1   (  0,  -1,   0) -7.2526   ( 1.435, -1.435, -1.435)  2.485
J_iso: -7.2526
----------------------------------------------------------------------------------------
   Fe1   Fe1   (  0,   0,  -1) -13.1577   ( 1.435,  1.435, -1.435)  2.485
J_iso: -13.1577
----------------------------------------------------------------------------------------
   Fe1   Fe1   (  0,   0,   1) -13.1577   (-1.435, -1.435,  1.435)  2.485
J_iso: -13.1577
----------------------------------------------------------------------------------------
   Fe1   Fe1   (  0,   1,   0) -7.2526   (-1.435,  1.435,  1.435)  2.485
J_iso: -7.2526
----------------------------------------------------------------------------------------
   Fe1   Fe1   (  1,  -1,   1) -12.5034   ( 1.435, -1.435,  1.435)  2.485
J_iso: -12.5034
----------------------------------------------------------------------------------------
   Fe1   Fe1   (  1,   0,   0) -9.9017   ( 1.435,  1.435,  1.435)  2.485
J_iso: -9.9017

**Comment by mailhexu** [comment]
Hi, 
Could you share the wannier input files? 
For bcc Fe it is a bit tricky, as the MLWF's will not be atomic centered.  There are some more issues with the bccFe that the Wannier functions have tails which could be extended to the neighboring Fe atoms. From my test using other basis set like the LCAO's in Siesta gives better result. 
There is an example for bcc Fe with QE:
https://github.com/mailhexu/TB2J_examples/tree/master/Wannier/bccFe_QE_Wannier90

Best regards,
HeXu

**Reply by chelman1** [reply]
I paste the w90 win file to produce the corresponding wannier. I use the site_symmetry option in order to keep the d-type of the selected bands.

num_bands         =   14
num_wann          =   5

bands_plot        = true
dis_win_max       = 30.0d0
dis_win_min       = 0.0
!dis_froz_max      =  20.0d0
!dis_froz_min      = -40.0
dis_num_iter      =  5000

num_iter          = 1400
mp_grid           = 8 8 8

!restart  = plot
write_hr = .true.
write_xyz = .true.
translate_home_cell = .true.
site_symmetry  = .true.
symmetrize_eps = 1d-9

dis_mix_ratio = 1.0

begin atoms_frac
Fe  0.000  0.000  0.000
end atoms_frac

begin projections
f= 0.0, 0.0, 0.0: d
end projections
begin unit_cell_cart
bohr
 2.71175  2.71175 2.71175
-2.71175  2.71175 2.71175
-2.71175 -2.71175 2.71175
end unit_cell_cart

Greetings

**Comment by mailhexu** [comment]
Could you check the wannier center and the wannier spreads to see if they are really symmetric?
And can you post both the DFT and Wannier band structrure? It is surprising to me that with only the d orbital projectors you can get very good wannier bands for bcc Fe. Usually s and p orbitals are needed too. 
Best regards,
HeXu

**Comment by chelman1** [comment]
ok, after recheck the band plot, I use a larger path in reciprocal space and I found discrepancies. 
Thanks for your help. I will try what you suggest
Regards 
Christian
